sklearn/experimental/enable_hist_gradient_boosting.py

```diff
@@ -26,7 +26,11 @@
 
 from .. import ensemble
 
-ensemble.HistGradientBoostingClassifier = HistGradientBoostingClassifier
-ensemble.HistGradientBoostingRegressor = HistGradientBoostingRegressor
+ensemble.HistGradientBoostingClassifier = (
+    HistGradientBoostingClassifier  # type: ignore
+)
+ensemble.HistGradientBoostingRegressor = (
+    HistGradientBoostingRegressor  # type: ignore
```

Comment:
Errors with a message that module doesn't include `HistGradientBoostingRegressor`.